Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseAutonomous.java

```diff
@@ -76,51 +91,124 @@ public void turnToAngle(double targetAngle) {
 
             // robot is turning counter-clockwise
             if (angleError > 0) {
-                motorPower = Math.min(angleError / -180.0, -0.2);
+                motorPower = Math.min(angleError / -250.0, -Constants.MINIMUM_TURNING_POWER);
 
             // robot is turning clockwise
             } else {
-                motorPower = Math.max(angleError / -180.0, 0.2);
+                motorPower = Math.max(angleError / -250.0, Constants.MINIMUM_TURNING_POWER);
             }
 
             motorFL.setPower(motorPower);
             motorFR.setPower(-motorPower);
             motorBL.setPower(motorPower);
             motorBR.setPower(-motorPower);
-
-            telemetry.addData("current", currentAngle);
-            telemetry.addData("error", angleError);
-            telemetry.addData("power", motorPower);
-            telemetry.update();
         }
 
-        motorFL.setPower(0.0);
-        motorFR.setPower(0.0);
-        motorBL.setPower(0.0);
-        motorBR.setPower(0.0);
+        stopDriveMotors();
     }
 
-    public void driveSlidesToPosition(int position) {
-        int error = position - motorLeftSlides.getCurrentPosition();
+    /**
+     * this method will allow the slides to move to a specified target position
+     * @param targetPosition target position for slides motors in ticks
+     */
+    public void driveSlidesAutonomous(int targetPosition) {
+        int error = targetPosition - motorLeftSlides.getCurrentPosition();
+        double motorPower;
 
-        while (Math.abs(error) > 25 && opModeIsActive()) {
+        // while slides aren't at target position
+        while (Math.abs(error) > Constants.ROBOT_SLIDE_TOLERANCE_TICKS && opModeIsActive()) {
+            error = targetPosition - motorLeftSlides.getCurrentPosition();
+            motorPower = error * Constants.SLIDE_MOTOR_KP;
+
+            // slides going down - full speed
```

Comment:
-0.75 isn't full speed

Reply:
changed to -1

Reply:
Are you sure you want to go down at full speed?

Reply:
This has likely been resolved, satvik should confirm

Reply:
We want to go at full speed because we need to save as much time as possible so we can cycle more cones. The motors were bought at the same time and are geared the same (also connected to the same axle), so there shouldn't be any problems with the motors being weird. We also run the motors without encoders so all 12 volts are being fed to the motors instead of the encoders using the built in PID loop to control the motors.